samples/cpp/text2image/CMakeLists.txt

```diff
@@ -45,3 +45,22 @@ install(TARGETS lora_stable_diffusion
         RUNTIME DESTINATION samples_bin/
         COMPONENT samples_bin
         EXCLUDE_FROM_ALL)
+
+# create txt2image_from_subcomponent sample executable
+
+add_executable(txt2image_from_subcomponent
```

Comment:
could you please add running this sample in GHA https://github.com/openvinotoolkit/openvino.genai/blob/master/.github/workflows/lcm_dreamshaper_cpp.yml#L73 instead of `samples/cpp/text2image/stable_diffusion` ? (as `samples/cpp/text2image/stable_diffusion` is validated here https://github.com/openvinotoolkit/openvino.genai/blob/master/.github/workflows/stable_diffusion_1_5_cpp.yml#L74)

Reply:
addressed in this PR: https://github.com/openvinotoolkit/openvino.genai/pull/1172